src/index/vespa_.py

```diff
@@ -350,38 +349,27 @@ def _get_vespa_instance() -> Vespa:
         cert=cert_location,
     )
 
-@retry(retry=retry_if_exception_type(VespaIndexError), wait=wait_exponential(multiplier=3), stop=stop_after_attempt(5))
+
+def _handle_feed_error(response: VespaResponse, id: str):
```

Comment:
Return type and doc string? 

Assuming that `response.is_successful()` is a subset of all error types or we may as well not have the `retry_if_exception_type`

Reply:
Updated